### DISCUSSION TITLE
You can add settings Can I get monaco-editor?

### DISCUSSION THREAD

**Discussion by watchakorn-18k** [DustinBrett/daedalOS#352] [Feature Requests]
Such as mouseWheelZoom: true, fontSize: 20, etc.

**Comment by DustinBrett** [comment]
Are you saying adding a config file for Monaco? If so I can for sure look into it. Atm I am unsure if it can be easily done as I don't think it works similar to VS Code in this way.

Currently I only have the command palete. I will add this to my todo feature list.

![image](https://github.com/DustinBrett/daedalOS/assets/16656318/e632830e-137a-40e2-bed7-4b929810f744)

![image](https://github.com/DustinBrett/daedalOS/assets/16656318/3e2a374b-22d1-440a-bde1-9ab713c8c27e)

**Reply by watchakorn-18k** [reply]
Thanks 🥳
